src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -242,19 +251,19 @@ def set_timesteps(self, num_inference_steps: int = None, device: Union[str, torc
             )
 
         sigmas = np.array(((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5)
+        log_sigmas = np.log(sigmas)
+
         if self.config.use_karras_sigmas:
-            log_sigmas = np.log(sigmas)
+            sigmas = np.flip(sigmas).copy()
```

Comment:
reversed order of `sigmas` here
fix https://github.com/huggingface/diffusers/issues/5077#issuecomment-1723931694